frontend/src/metabase-types/api/dataset.ts

```diff
@@ -94,7 +94,7 @@ export interface NativeQueryForm {
 
 export type SingleSeries = {
   card: Card;
-} & Dataset;
+} & Pick<Dataset, "data" | "error">;
```

Comment:
At runtime this value seems to only ever be `{ card, data }`

![Screenshot 2023-10-17 at 3.10.14 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/e9367628-87d8-4159-b47d-fcf1310e31c0/Screenshot%202023-10-17%20at%203.10.14%20PM.png)

In the codebase there was only a single reference to any other property, `error`

![Screenshot 2023-10-17 at 7.10.13 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/716e0911-5464-42f5-886b-8b1af84d3dcb/Screenshot%202023-10-17%20at%207.10.13%20PM.png)

Removing the extra types saves us from needing to specify them in mock data for tests.

Reply:
Good! I pushed this change to the base branch because I also need this